Monika-After-Story/MonikaModDev
# Better Stories System

Stories now use a more dynamic system. Featuring less redundant code and actually working return keys for *`A new story`* (#6637)

# Testing:
- Verify stories still work properly
- Verify unlocking stories still works properly
- Verify return keys work on stories accessed via `A new story`
